io.cpp

```diff
@@ -80,11 +80,21 @@ int main()
         {
             for(int x = 0; x < S_WIDTH; x++)
             {
-                // Red color
-                framePtr[y][x] = 0x00ff0000;
+                // Black color
+                framePtr[y][x] = 0xff000000;
             }
         }
 
+	// Draw a small box
+	for(int y = 50; y < 100; y++)
+	  {
+	    for(int x = 50; x < 100; x++)
+	      {
+		framePtr[y][x] = 0x0ffffffff;
+	      }
+	  }
+
```

Comment:
Why did you choose 50x50? Why not something a bit bigger?

Reply:
I think because my monitor is a small resolution it seemed normal. Will make bigger for future projects though.